3rdparty/pine/pine_server.h

```diff
@@ -381,7 +381,11 @@ namespace pine
 
 			m_msgsock = accept(m_sock, 0, 0);
 
+#ifdef _WIN32
```

Comment:
Might as well make a constant at the top of the file and use it for both. The debt doesn't feel worth it to just allow use of a magic number.

Reply:
The problem is a compilation error because the result of `accept` is signed on Linux and unsigned on Windows.

```
C:/src/rpcs3/3rdparty/pine/pine_server.h:384:39: error: comparison of integer expressions of different signedness: 'pine::SOCKET' {aka 'long long unsigned int'} and 'int' [-Werror=sign-compare]
  384 |                         if (m_msgsock == -1)
      |                             ~~~~~~~~~~^~~~~
```

I could disable the warning with a pragma in this file.

Reply:
What he's saying is that you could do something like this:
```cpp
#ifdef _WIN32
constexpr SOCKET invalid_socket = INVALID_SOCKET;
#else
constexpr int invalid_socket = -1;
#endif
```
and then just do comparisons to this `invalid_socket` value.